packages/meditrak-server/src/devops/CreateThumbnail/CreateThumbnail.js

```diff
@@ -1,33 +1,35 @@
+/* eslint-disable no-console */
```

Comment:
Lambda may not support es6, I guess that's why this was all `var`

Reply:
We use Node.js 8.10 for this Lambda and full `const` support was added in Node 6.4, but I agree that it's probably not worth the risk changing this!

Reply:
Have reverted code changes to minimise regression risk... Just added some eslint directives here